grafana-plugin/src/components/UserGroups/UserGroups.styles.ts

```diff
@@ -10,8 +10,8 @@ export const getUserGroupStyles = (theme: GrafanaTheme2) => {
 
     sortable: css`
       z-index: 1062;
-      box-shadow: var(--focused-box-shadow);
-      background: var(--hover-selected-hardcoded) !important;
+      box-shadow: ${theme.isDark ? 'rgb(17 18 23) 0 0 0 2px, rgb(61 113 217) 0 0 0 4px;' : ''};
+      background: ${theme.isDark ? '#34363d' : ''};
```

Comment:
Shouldn't it be some var from theme?

Reply:
These are custom made for oncall, they're not in core grafana

Reply:
Can we put them in some global file and import? At least the background color. So it's easier to track what custom colors we have in app and reuse if needed